python/lsst/ts/watcher/config_schema.py

```diff
@@ -105,26 +105,9 @@
           minItems: 1
           items:
             type: string
-        responders:
-          description: ->
-            Teams, users, and/or schedules to which to escalate these alarms.
-            The format is the same used by OpsGenie to create an alert.
-          type: array
-          items:
-            type: object
-            required: [name, type]
-            additionalProperties: false
-            minItems: 1
-            properties:
-              name:
-                description: ->
-                    Name of team, user, or schedule to which to escalate this alarm.
-                    Note that IDs are not supported; it must be a name.
-                type: string
-              type:
-                description: Type of name.
-                type: string
-                enum: ["team", "user", "schedule"]
+        responder:
+          description: Who to escalate this alarm to.
```

Comment:
In `alarm.py` the description is `Who or what to escalate the alarm to.` so maybe use that here as well for consistency?